src/main/java/seedu/address/logic/parser/CommentCommandParser.java

```diff
@@ -20,8 +22,14 @@ public class CommentCommandParser implements Parser<CommentCommand> {
      */
     public CommentCommand parse(String args) throws ParseException {
         requireNonNull(args);
+
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_COMMENT);
+
```

Comment:
The addition of arePrefixesPresent method enhances the validation logic, but ensure it is covered with tests.